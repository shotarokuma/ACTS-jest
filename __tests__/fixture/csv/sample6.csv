Column1	Column2	Column3	Column4	Column5	Column6	Column7	Column8	Column9	Column10	Column11	Column12	Column13	Column14	Column15	Column16	Column17	Column18	Column19	Column20
sample cell
	sample cell
	sample cell
	sample cell
	sample cell
	sample cell
	sample cell
	sample cell
	sample cell
	sample cell
	sample cell
	sample cell
	sample cell
	sample cell
	sample cell
	sample cell
	sample cell
	sample cell
	sample cell
	sample cell
