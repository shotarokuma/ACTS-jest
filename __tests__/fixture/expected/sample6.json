[
{"Column1":"sample cell"},
{"Column1":"","Column2":"sample cell"},
{"Column1":"","Column2":"sample cell"},
{"Column1":"","Column2":"sample cell"},
{"Column1":"","Column2":"sample cell"},
{"Column1":"","Column2":"sample cell"},
{"Column1":"","Column2":"sample cell"},
{"Column1":"","Column2":"sample cell"},
{"Column1":"","Column2":"sample cell"},
{"Column1":"","Column2":"sample cell"},
{"Column1":"","Column2":"sample cell"},
{"Column1":"","Column2":"sample cell"},
{"Column1":"","Column2":"sample cell"},
{"Column1":"","Column2":"sample cell"},
{"Column1":"","Column2":"sample cell"},
{"Column1":"","Column2":"sample cell"},
{"Column1":"","Column2":"sample cell"},
{"Column1":"","Column2":"sample cell"},
{"Column1":"","Column2":"sample cell"},
{"Column1":"","Column2":"sample cell"}
]
